postgresql-96/pg_comparator.spec

```diff
@@ -123,5 +123,9 @@ rm -rf %{buildroot}
 ###############################################################################
 
 %changelog
+* Wed May 11 2017 Andrey Kulikov <avk@brewkeeper.net> - 2.2.6-0
```

Comment:
`Wed May 11 2017` is invalid date. Use command `date +'%a %b %d %Y'` for generating valid date for changelog.